erdblick_app/app/parameters.service.ts

```diff
@@ -155,19 +210,19 @@ const erdblickParameters: Record<string, ParameterDescriptor> = {
         urlParam: true
     },
     osm: {
-        converter: val => val === 'true',
+        converter: val => val === 'true' || val === '1',
         validator: val => typeof val === 'boolean',
         default: true,
         urlParam: true
     },
     layers: {
-        converter: val => JSON.parse(val),
+        converter: val => decodeCompactBooleans(JSON.parse(val), 'layers'),
         validator: val => Array.isArray(val) && val.every(item => Array.isArray(item) && item.length === 4 && typeof item[0] === 'string' && typeof item[1] === 'number' && typeof item[2] === 'boolean' && typeof item[3] === 'boolean'),
         default: [],
         urlParam: true
     },
     styles: {
-        converter: val => JSON.parse(val),
+        converter: val => decodeCompactBooleans(JSON.parse(val), 'styles'),
         validator: val => {
             return typeof val === "object" && Object.entries(val as Record<string, ErdblickParameters>).every(
                 ([_, v]) => validateObject({visible: "boolean", showOptions: "boolean", options: "object"})(v));
```

Comment:
Probably, we should also shorten the field names for the style config entries. This one is on me, you warned me 🥲